book_source/developers_guide/Adding-an-Input-Converter.Rmd

```diff
@@ -0,0 +1,61 @@
+General Note: dates in the database should be datatime (preferably with timezone), and datetime passed around in PEcAn should be of type POSIXlt.
+
+# Met Data
+
+The standard met data inputs should be of the form:
+
+Converters from a raw to standard format go in `/modules/data.atmosphere/R`; converters from standard to model-specific go in `models/<mymodel>/R`.
+
+* For a number of common gridded products (NARR, CRUNCEP, ISIMIP), there are bash scripts for converting and rechunking in this repository: https://github.com/ebimodeling/model-drivers; these are not generalized, but may be merged into PEcAn at some point.
+
+Examples:
+* NARR:
+* CRUNCEP:
+* ISIMIP: 
+
+Names should be `met2CF.<sourcename>` and `met2model.<modelname>`.
+
+## Dimensions:
+
+|CF standard-name | units |
+|:------------------------------------------|:------|
+| time | days since 1700-01-01 00:00:00 UTC|
+| longitude | degrees_east|
+| latitude |degrees_north|
+
+## The variable names should be `standard_name`
+
+| CF standard-name                          | units | bety         | isimip       | cruncep | narr  | ameriflux |
```

Comment:
We should replace this with R code to fetch the relevant table from BETY and render it using `knitr::kable()`.